src/auth.py

```diff
@@ -5,84 +5,95 @@
 from urllib.parse import urljoin
 
 import requests
+from beartype import beartype
 
 from .config import get_value as get_config
 from .exceptions import DeepOriginException
 
 __all__ = [
-    "get_do_api_tokens",
-    "cache_do_api_tokens",
-    "read_cached_do_api_tokens",
-    "remove_cached_do_api_tokens",
-    "sign_into_do_platform",
-    "refresh_access_to_do_platform",
+    "get_tokens",
+    "cache_tokens",
+    "read_cached_tokens",
+    "remove_cached_tokens",
+    "authenticate",
+    "refresh_tokens",
 ]
 
 
+@beartype
+def tokens_exist() -> bool:
+    """Check if the cached API tokens exist"""
+    config = get_config()
+    return os.path.isfile(config.api_tokens_filename)
+
+
 @functools.cache
-def get_do_api_tokens(verbose: bool = False) -> tuple[str, str]:
-    """Get a token for accessing the Deep Origin API
+def get_tokens(
```

Comment:
this is the primary way we interact with the auth module. in almost every case, it should be sufficient to simply `get_tokens` and simply use that. the complexities of caching, signing in, etc., are all abstracted away

Reply:
I'm onboard with simplifying this.

Before I was trying to limit caching to keep alive the possibility of using this in a ComputeBench shared with multiple users, at least one of whom has root access. I think this was at least premature:
- First of all, in most cases there are better ways to facilitate collaboration than to share benches. 
- Second, sharing benches is a distant goal. 
- Third, to do bench sharing properly, I think only a special class of admin users should be able to have root access, which would enable them to see other users stored keys.